operations/cancellationpolicies.md

```diff
@@ -44,44 +46,45 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
 | `EnterpriseIds` | array of string | optional, max 1000 items | Unique identifiers of the [Enterprises](enterprises.md#enterprise). If not specified, the operation returns data for all enterprises within scope of the Access Token. |
-| `ServiceIds` | array of string | required, max 1000 items | Unique identifiers of the [Service](services.md#service). |
-| `CancellationPolicyIds` | array of string | optional, max 1000 items | Unique identifiers of the [Cancellation Policy](#cancellationpolicy). Required if no other filter is provided. |
+| `ServiceIds` | array of string | required, max 100 items | Unique identifiers of the [Service](services.md#service). |
```

Comment:
I guess change of max items from 1000 to 100 is intentional and code could be considered as source of truth